app/src/main/java/edu/uc/app/parkfinder/service/ParkService.kt

```diff
@@ -22,6 +23,7 @@ class ParkService {
              */
             override fun onFailure(call: Call<ArrayList<Park>>, t: Throwable) {
                 // TODO: fill this in
+                Log.d("Park ArrayList Callback", "Something went wrong!", t)
```

Comment:
Added starter debug log statement to onFailure(). Came in a lot of use when endpoints/connections were not operating how I needed!

Reply:
Ah, log statements are always a good idea.

In this case, I'd go with Log.e (error) instead of Log.d (debug), since you're in a failure handler.

Reply:
good point, should have changed that after debugging!